doc/release-notes/7700-upgrade-payara.md

```diff
@@ -0,0 +1,19 @@
+### Payara 5.2021.3 (or Higher) Required
+
+Some changes in this release require an upgrade to Payara 5.2021.3 or higher.
+
+Instructions on how to update can be found in the
+[Payara documentation](https://docs.payara.fish/community/docs/5.2021.3/documentation/user-guides/upgrade-payara.html)
+
+It would likely be safer to upgrade Payara first, while still running Dataverse 5.4, and then proceed with the steps
```

Comment:
@poikilotherm can you change this to 5.5 though?

Reply:
Done